website/client/components/challenges/myChallenges.vue

```diff
@@ -117,44 +122,86 @@ export default {
       ],
       search: '',
       filters: {},
+      page: 0,
     };
   },
   mounted () {
     this.loadChallenges();
   },
   computed: {
+    ...mapState({user: 'user.data'}),
     filteredChallenges () {
-      let search = this.search;
-      let filters = this.filters;
-      let user = this.$store.state.user.data;
+      const filters = this.filters;
+      const user = this.user;
+
+      return this.challenges.filter(challenge => {
+        let isMember = true;
 
-      // @TODO: Move this to the server
-      return this.challenges.filter((challenge) => {
-        return this.filterChallenge(challenge, filters, search, user);
```

Comment:
why isn't this function used anymore? also it looks like it was only used here so if it's really not needed it can be removed from the mixin?

Reply:
most of it is in the server now, the function will be removed